src/Mod/Draft/draftobjects/patharray.py

```diff
@@ -95,27 +120,32 @@ def execute(self,obj):
         import Part
         import DraftGeomUtils
         if obj.Base and obj.PathObj:
-            pl = obj.Placement
+            pl = obj.Placement                  #placement of whole pathArray
             if obj.PathSubs:
                 w = self.getWireFromSubs(obj)
             elif (hasattr(obj.PathObj.Shape,'Wires') and obj.PathObj.Shape.Wires):
                 w = obj.PathObj.Shape.Wires[0]
             elif obj.PathObj.Shape.Edges:
                 w = Part.Wire(obj.PathObj.Shape.Edges)
             else:
-                App.Console.PrintLog ("_PathArray.createGeometry: path " + obj.PathObj.Name + " has no edges\n")
+                Gui.Console.PrintLog ("PathArray.execute: path " + obj.PathObj.Name + " has no edges\n")
```

Comment:
This should be `App.Console`, not `Gui.Console`.

Reply:
Done.  Probably been like that for years.